### ISSUE TITLE
Recurrence problem of Youtube-vis2019

### ISSUE THREAD

**Issue by kamdkslansgka** [zhang-tao-whu/DVIS_Plus#31]
Hello author, thank you for your excellent work.
I would like to ask, I can't seem to reproduce the accuracy in the paper in my experiment. The accuracy of Youtube-2019 in the paper is 55.5 (Resnet50), but I reproduced it and it is only 51. The following is the log of my training. Is there any problem?

[log.txt](https://github.com/user-attachments/files/20146768/log.txt)

The training script is as follows：

python /data/Share/Student/2024/ChenYuran/experiment/DVIS_Plus-main/DVIS_Plus/train_net_video.py \
    --num-gpus 8 \
    --config-file /data/Share/Student/2024/ChenYuran/experiment/DVIS_Plus-main/DVIS_Plus/configs/dvis_Plus/ytvis19/DVIS_Plus_Online_R50_8wIter.yaml \
    --resume \
    MODEL.WEIGHTS /data/Share/Student/2024/ChenYuran/experiment/DVIS_Plus-main/DVIS_Plus/ctvis_r50_ytvis19.pth \
    SOLVER.IMS_PER_BATCH 8
